### DISCUSSION TITLE
How to use OLC

### DISCUSSION THREAD

**Discussion by gmeader** [mikacousin/olc#17] [Q&A] [answered]
I don't know how to start using OLC. I have a Congo manual. But it does not help get started with OLC.
I found the Keyboard shortcuts in the menu.
1. I have the Patch Outputs tab visible. It is empty except for two buttons.  I click Patch 1:1, but nothing happens. Should something be displayed?
2. . In the channels tab, nothing is displayed. What do I do to get some channels displayed, and how can I set the levels? 
Typing 1C100@ does nothing.
3. in Sequences, I found that if I type a number and a capital R, a new cue number is added. but since I have no channels with levels, nothing useful is stored.
4. The Virtual console window is blank.

Maybe you can make a "getting started" document or video?

UPDATE:  I found some documentation here: http://mikacousin.github.io/olc/  That link needs to be placed in the README.md here.

**Comment by mikacousin** [comment, marked as answer]
Hi !

Thanks, I added the link to the README.md.

According to https://github.com/mikacousin/olc/issues/18, the software seems not to work normally for you. Let's fix the problem and we'll see how to use it next.
